client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -415,6 +420,18 @@ const PatternAssembler = ( props: StepProps & NoticesProps ) => {
 	} );
 
 	const getBackLabel = () => {
+		// Exits the Assembler.
+		if (
+			isSiteAssemblerFlow( flow ) &&
+			navigator.location.path?.startsWith( NAVIGATOR_PATHS.MAIN )
```

Comment:
[nit] Now we can use the `currentScreen.previousScreen` to determine whether it is the first screen.